ansible_collections/arista/avd/plugins/modules/eos_validate_state_runner.py

```diff
@@ -65,15 +60,15 @@
           - An optional list of specific tests in the category to skip (e.g., `VerifyRoutingProtocolModel` in `AvdTestBGP`).
           - If not specified, all tests in the category are considered.
           - For a complete list of available tests,
-            see [link to the test list](https://avd.sh/en/stable/roles/eos_validate_state/ANTA-Preview.html#test-categories).
+            see [link to the test list](https://avd.sh/en/stable/roles/eos_validate_state/anta_integration.html#test-categories).
```

Comment:
let's not use `avd.sh`
```suggestion
            see [link to the test list](https://avd.arista.com/stable/roles/eos_validate_state/anta_integration.html#test-categories).
```